trajoptlib/src/RustFFI.cpp

```diff
@@ -645,8 +645,9 @@ void DifferentialTrajectoryGenerator::add_callback(
         for (const auto& cppSample : cppTrajectory.samples) {
           rustSamples.push_back(DifferentialTrajectorySample{
               cppSample.timestamp, cppSample.x, cppSample.y, cppSample.heading,
-              cppSample.velocityL, cppSample.velocityR, cppSample.accelerationL,
-              cppSample.accelerationR, cppSample.forceL, cppSample.forceR});
+              cppSample.velocityL, cppSample.velocityR, cppSample.omega,
```

Comment:
`angularVelocity` should be used here instead since none of the others use Greek letter names.